### PR TITLE
wording changes

### PR DESCRIPTION
**Pull request by pmechineniesac**
https://jira.hl7.org/browse/FHIR-45466

### CODE REVIEW

File: input/pages/measure-conformance.md [hunk 1]
```diff
@@ -761,7 +761,7 @@ In addition, the formula for calculating the measure score is implied by the sco
...
+In addition to the measure scoring, measures generally fall into two categories, patient-based, and episode-based (e.g. encounter-based). In general, patient-based measures count the number of patients in each population, while episode-based measures count the number of items (such as encounters) in each population. Although the calculation formulas are conceptually the same for both categories, for ease of expression, population criteria for patient-based measures indicates whether a patient matches the population criteria (true) or not (false). Episode-based measures return the item to be counted such as an encounter or procedure.
```
Comment (p9g) [inline input/pages/measure-conformance.md:764]: Line 866 should also change "For a complete example of an **encounter-based** proportion measure," to "For a complete example of an **episode-of-care** proportion measure,"

File: input/pages/measure-conformance.md
Comment (yanheras) [inline input/pages/measure-conformance.md:1]: This tracker was re-opened on 7/19/2024 and re-disposed. Please make changes based on the new updated resolution (https://jira.hl7.org/browse/FHIR-45466)

**Review by yanheras — CHANGES_REQUESTED** [review]

File: input/pages/measure-conformance.md [hunk 2]
```diff
@@ -863,7 +863,7 @@ The referenced CQL expressions return either an indication that a patient meets
 
 In Table 3-2, the first measure is an example of a patient-based measure. Each patient may contribute at most one count to the denominator and numerator, regardless of how many encounters they had. The second measure is an episode-of-care measure where each patient may contribute zero or more encounters to the denominator and numerator counts.
 
-For complete examples of patient-based proportion measures, see the Screening Measure [Examples](examples.html). For a complete example of an encounter-based proportion measure, see the [EXM108](Measure-EXM108-FHIR.html) measure included in this implementation guide.
+For complete examples of non-patient based proportion measures, see the Screening Measure [Examples](examples.html). For a complete example of an encounter-based proportion measure, see the [EXM108](Measure-EXM108-FHIR.html) measure included in this implementation guide.
```
Comment (yanheras) [inline input/pages/measure-conformance.md:866]: 1) do not change "patient-based" to "non-patient based"
2) The resolution says "Throughout QMs section, change all instances of 'episode-of-care' or 'encounter-based' to 'non-patient based'
- To apply the changes according to this resolution, it needs to search the entire "measure-conformance.md" file and change all instances of "episode-of-care" based measures and "encounter-based" measures to "non-patient based" measures.

**Review by yanheras — CHANGES_REQUESTED** [review]

**Comment by brynrhodes** [comment]
Need to be reviewed after this PR is merged: https://github.com/HL7/cqf-measures/pull/278

**Review by brynrhodes — CHANGES_REQUESTED** [review]
Needs to be re-reviewed after this PR is merged: https://github.com/HL7/cqf-measures/pull/278

**Comment by yanheras** [comment]
made a few edits and merged.

**Review by yanheras — APPROVED** [review]

**Review by brynrhodes — APPROVED** [review]

**Pull request merged**